src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1667,6 +1667,41 @@ class Simplex_tree {
     return sh; // None of its faces has the same filtration.
   }
 
+ public:
+  /** \brief This function resets filtration value from a given dimension. Resets all the Simplex_tree when
+   * `min_dim = 0`.
+   * @param[in] filt_value The new filtration value.
+   * @param[in] min_dim The minimal dimension.
+   */
+  void reset_filtration(Filtration_value filt_value, int min_dim) {
```

Comment:
Default value of 0 for min_dim?

Reply:
Done on 12371aa8
Do you think we should default `filt_value` (to `NaN` for instance) ?

Reply:
> Do you think we should default filt_value (to NaN for instance) ?

As you like.

Reply:
ok let it like that.